exporters/zipkin/src/main/java/io/opentelemetry/exporter/zipkin/ZipkinSpanExporter.java

```diff
@@ -141,7 +141,9 @@ static Span generateSpan(SpanData spanData, Endpoint localEndpoint) {
         });
     SpanData.Status status = spanData.getStatus();
     // for GRPC spans, include status code & description.
-    if (status != null && spanAttributes.get(SemanticAttributes.RPC_SERVICE) != null) {
+    if (status != null
+        && !status.isUnset()
+        && spanAttributes.get(SemanticAttributes.RPC_SERVICE) != null) {
```

Comment:
I'm reading the attached issue https://github.com/open-telemetry/opentelemetry-specification/pull/1186/files and can't find anything about this check, can you point me at it? IIUC we're supposed to set status even if not rpc.

Reply:
> can't find anything about this check

So do I 
Can you help to confirm this? @Oberon00

Reply:
This was definitely a holdover from when the statuses were so closely aligned with the GRPC status codes. Should be fine, I would think, to remove it. 👍

Reply:
Thanks. It was removed.